src/app/Dashboard/Dashboard.tsx

```diff
@@ -1,10 +1,40 @@
 import * as React from 'react';
 import { PageSection, Title } from '@patternfly/react-core';
 
-const Dashboard: React.FunctionComponent = () => (
+const Dashboard: React.FunctionComponent = () => {
+
+  const [currentInput, setCurrentInput] = React.useState('')
+  const [list, setList] = React.useState([]);
+
+  const AddTodo = (event) => {
+    event.preventDefault();
+    // const newList = list;
+    // newList.push(currentInput);
+    setList([...list, currentInput]);
+    console.log(list);
+  }
+  return (
   <PageSection>
     <Title headingLevel="h1" size="lg">Dashboard Page Title!</Title>
+    <h1>TODO APP</h1>
+<form className="input-box">
+<input className="input" placeholder="enter you name" onChange={(event)=>{
```

Comment:
The input field should have an initial value, 
```
- <input className="input" placeholder="enter you name">
+ <input className="input" placeholder="enter you name" value={currentInput}/>
``` otherwise it will become uncontrolled input